src/smart-components/order/order-item.js

```diff
@@ -1,155 +1,121 @@
-                      <CatalogLink
-                        pathname={ORDER_ROUTE}
-                        searchParams={searchParams}
-                      >
-                        {formatMessage(ordersMessages.compositeTitle, {
-                          name: orderName,
-                          id: item.id
-                        })}
-                      </CatalogLink>
-                    </Text>
-                  </LevelItem>
-                  <LevelItem className="flex-item-no-wrap">
-                    <CatalogLink
-                      pathname={routeMapper[item.state] || ORDER_ROUTE}
-                      searchParams={searchParams}
-                    >
-                      {item.state === 'Failed' && (
-                        <ExclamationCircleIcon className="pf-u-mr-sm icon-danger-fill" />
-                      )}
-                      {formatMessage(statesMessages[translatableState])}
-                    </CatalogLink>
-                  </LevelItem>
-                </Level>
-              </GridItem>
-              <GridItem>
-                <Level>
-                  <LevelItem>
-                    <Text className="pf-u-mb-0" component={TextVariants.small}>
-                      {formatMessage(statesMessages.ordered)}
-                      &nbsp;
-                      <DateFormat date={item.created_at} variant="relative" />
-                    </Text>
-                  </LevelItem>
-                  <LevelItem>
-                    <Text className="pf-u-mb-0" component={TextVariants.small}>
-                      {formatMessage(ordersMessages.orderedBy, {
-                        owner: item.owner
-                      })}
-                    </Text>
-                  </LevelItem>
-                  <LevelItem>
-                    <Text className="pf-u-mb-0" component={TextVariants.small}>
-                      {formatMessage(ordersMessages.lastUpdated)}
-                      &nbsp;
-                      <DateFormat
-                        date={
-                          item.orderItems[0] && item.orderItems[0].updated_at
-                        }
-                        variant="relative"
-                      />
-                    </Text>
-                  </LevelItem>
-                </Level>
-              </GridItem>
-            </Grid>
-          </TextContent>
-        </TableCell>
-      </tr>
-    );
-  },
-  (prevProps, nextProps) => prevProps.id === nextProps.id
-);
-
-OrderItem.propTypes = {
-  item: PropTypes.object.isRequired
+        </Fragment>
+      )
+    },
+    item.orderName,
+    item.owner,
+    {
+      title: (
+        <Text className="pf-u-mb-0" component={TextVariants.small}>
+          <DateFormat date={item.created_at} variant="relative" />
+        </Text>
+      )
+    },
+    {
+      title: (
+        <Text className="pf-u-mb-0" component={TextVariants.small}>
+          {formatMessage(ordersMessages.lastUpdated)}
+          &nbsp;
```

Comment:
Can we put classname with a modifier to the next DateFormat?

Reply:
No need the last updated string should be removed!